src/features/Chat/components/recipe/RecipeLetter.tsx

```
@@ -154,7 +154,7 @@ export function RecipeLetter({ recipe, onSave, isSaved }: RecipeLetterProps) {
         </div>
 
         <div className={cn(EYEBROW_BASE, "text-muted-foreground")}>
-          The way I make it
+          From Ah Mah's Kitchen
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)